maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -194,9 +200,24 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 				}
 			}
 
+			boolean isWab = PACKAGING_WAR.equals(project.getPackaging());
```

Comment:
We are now conflating `-wab` with packaging type = war. What if someone uses `-wab` with packaging type = jar?

Reply:
if someone uses -wab with jar they'll get messed up jar. Should I block the combination with an error?

Reply:
my last update solves this concern by throwing an error (like it previously did) when pairing `-wab` or `-wablib` with packaging `jar`.